src/core/js/adapt.js

```diff
@@ -30,7 +30,7 @@ class AdaptSingleton extends LockingModel {
    * @deprecated since v6.0.0 - please use `Adapt.store` instead
    */
   get componentStore() {
-    this.log && this.log.deprecated('Adapt.componentStore, please use Adapt.store instead');
+    this.log?.deprecated('Adapt.componentStore, please use Adapt.store instead');
```

Comment:
I do wonder if the check for `this.log` is even necessary?

Reply:
I also wonder. @oliverfoster?

Reply:
It's a quirk of the compilation order that makes `src/core/logging` load before that getter would ever be called. In practise it can be removed.

Reply:
as is, it's still causing an eslint error for some reason:
![image](https://user-images.githubusercontent.com/1249597/108093326-803e8e80-7075-11eb-991a-f5335bbaf2c6.png)

so yeah let's get rid of the check